src/main/java/org/broadinstitute/hellbender/utils/help/HelpConstants.java

```diff
@@ -65,7 +65,10 @@ public static String forumPost(String post) {
     public final static String DOC_CAT_VARDISC = "Variant Discovery Tools";
     public final static String DOC_CAT_VAREVAL = "Variant Evaluation Tools";
     public final static String DOC_CAT_VARMANIP = "Variant Manipulation Tools";
-
+    
+    public final static String DOC_CAT_ANNOTATORS = "Annotation Modules";
```

Comment:
I know we called these "Annotation Modules" in GATK3, but the word "module" seems out of place to me. I think everywhere else in the doc we just refer to them as annotations. My suggestion would be to just call them "Variant Annotations".

Reply:
Let's ask @vdauwera to chime in  here.

Reply:
works for me

Reply:
Making this change. Thanks for the suggestion!